File: Back/.Net/Modules/Bets/BetFriends.Bets.Infrastructure/Repositories/Sql/DataAccess/FriendshipEntity.cs
using BetFriends.Bets.Domain.Friendships;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BetFriends.Bets.Infrastructure.Repositories.Sql.DataAccess;

[Table("friendships", Schema = "bet")]
public class FriendshipEntity
{
    public FriendshipEntity()
    {
        
    }

    public FriendshipEntity(Friendship friendship)
    {
        FriendId = friendship.FriendId;
        RequesterId = friendship.RequesterId;
    }

    [Column("friend_id"), Required]
    public Guid FriendId { get; init; }
    [Column("requester_id"), Required]
    public Guid RequesterId { get; init; }

    [Column("created_at")]
    public DateTime CreatedAt { get; init; }
}

File: Back/NodeJS/modules/bets/src/infrastructure/DataAccess/entities/FriendshipEntity.ts
import { Column, Entity } from "typeorm";
import { Friendship } from "../../../domain/friendships/Friendship";

@Entity({
    schema: "bet",
    name: "friendships"
})

export class FriendshipEntity {
    static Create(friendship: Friendship): FriendshipEntity {
        const entity = new FriendshipEntity();
        entity.FriendId = friendship.FriendId.Value;
        entity.RequesterId = friendship.RequesterId.Value;
        return entity;
    }
    @Column({
        name: "friend_id"
    })
    FriendId: string;

    @Column({
        name: "requester_id"
    })
    RequesterId: string;

    @Column({
        name: "created_at"
    })
    CreatedAt: Date;
}